readthedocs/api/base.py

```diff
@@ -12,7 +12,7 @@
 from django.core.cache import cache
 from django.shortcuts import get_object_or_404
 from tastypie import fields
-from tastypie.authorization import DjangoAuthorization
+from tastypie.authorization import DjangoAuthorization, ReadOnlyAuthorization
```

Comment:
There are other resources using `DjangoAuthorization`, but tests were failing only for the `ProjectResource`

Reply:
I think we don't have tests for the other resources, that's why